libsolidity/parsing/DocStringParser.cpp

```diff
@@ -119,21 +119,17 @@ DocStringParser::iter DocStringParser::parseDocTagParam(iter _pos, iter _end)
 		return _end;
 	}
 	auto nameEndPos = firstSpaceOrTab(nameStartPos, _end);
-	if (nameEndPos == _end)
-	{
-		appendError("End of param name not found: " + string(nameStartPos, _end));
-		return _end;
-	}
 	auto paramName = string(nameStartPos, nameEndPos);
 
 	auto descStartPos = skipWhitespace(nameEndPos, _end);
-	if (descStartPos == _end)
+	auto nlPos = find(descStartPos, _end, '\n');
```

Comment:
Shouldn't this be `find(nameEndPos, _end, '\n')`?

Reply:
nlPos is the end of the description, so searching starts at the beginning of the description... descStartPos might be _end already, but that's ok - then nlPos will be _end as well and the error about a missing description will be triggered...